src/System.Management.Automation/engine/hostifaces/HostUtilities.cs

```diff
@@ -504,8 +504,17 @@ internal static List<string> GetSuggestion(HistoryInfo lastHistory, object lastE
         internal static string RemoveGuidFromMessage(string message, out bool matchPattern)
```

Comment:
The out param isn't even being used and should be removed

Reply:
Removed.